scss/pages/_inner.scss

```diff
@@ -0,0 +1,195 @@
+.post-content {
+    h1, h2, h3, h4, h5, h6 {
+        color: rgb(72, 73, 78);
+        font-weight: bold;
+        margin: 0 0 16px;
+    }
+
+    h1 {
+        font-size: 38px;
+        line-height: 1.369;
+    }
+
+    h2 {
+        font-size: 28px;
+        font-weight: 600;
+    }
+
+    a:not(.button) {
+        text-decoration: none;
+        color: $green-light;
+        &:hover {
+            text-decoration: underline;
+        }
+    }
+
+    table {
+        max-width: 100%;
+        margin-bottom: 20px;
+        border: 1px solid $grey-7;
+        border-radius: 4px;
+        text-align: left;
+        word-break: break-word;
+        color: $grey-8;
+        font-size: 16px;
+        line-height: 1.5;
+        font-weight: 500;
+    }
+
+    caption {
+        padding-top: 8px;
+        padding-bottom: 8px;
+        text-align: left;
+    }
+
+    th,
+    td {
+        padding: 10px 15px;
+        hyphens: auto;
+        min-width: 100px;
+        border: 1px solid $grey-7;
+    }
+
+    th {
+        color: rgb(153, 155, 170);
+        font-size: 13px;
+        font-weight: 600;
+    }
+
+    td {
+    }
+
+    /* customstyles.css*/
+    tr.success {
```

Comment:
Please rescructure these styles so everything will be in the `table` selector and you can use the parent selector.